docs/book/src/clusterctl/commands/alpha-rollout.md

```diff
@@ -0,0 +1,51 @@
+# clusterctl alpha rollout
+
+The `clusterctl alpha rollout` command manages the rollout of a cluster-api resource. It consists of several sub-commands which are documented below. 
```

Comment:
maybe we should make `cluster-api OR Cluster API`  word consistent throughout this document? (`Cluster API` seems to be more frequently used in the repo)